Gemfile.lock

```diff
@@ -1,78 +1,78 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.0)
-      actionview (= 6.0.0)
-      activesupport (= 6.0.0)
-      rack (~> 2.0)
+    actionpack (6.0.2.1)
+      actionview (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+      rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.0)
-      activesupport (= 6.0.0)
+    actionview (6.0.2.1)
+      activesupport (= 6.0.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.0)
-      activesupport (= 6.0.0)
-    activerecord (6.0.0)
-      activemodel (= 6.0.0)
-      activesupport (= 6.0.0)
-    activesupport (6.0.0)
+    activemodel (6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activerecord (6.0.2.1)
+      activemodel (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activesupport (6.0.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.1, >= 2.1.8)
-    builder (3.2.3)
+      zeitwerk (~> 2.2)
+    builder (3.2.4)
     concurrent-ruby (1.1.5)
-    crass (1.0.4)
+    crass (1.0.5)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.8.0)
-    i18n (1.6.0)
+    erubi (1.9.0)
+    i18n (1.7.0)
       concurrent-ruby (~> 1.0)
-    json (2.2.0)
+    json (2.3.0)
     jwt (2.1.0)
     jwt_signed_request (2.5.1)
       jwt (>= 1.5.0, < 2.2.0)
       rack
-    loofah (2.2.3)
+    loofah (2.4.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     method_source (0.9.2)
     mini_portile2 (2.4.0)
-    minitest (5.11.3)
-    money (6.13.4)
+    minitest (5.13.0)
```

Comment:
### [minitest](https://github.com/seattlerb/minitest)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 5.11.3 → 5.13.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **46** commits. See the full changes on [the compare page](https://github.com/seattlerb/minitest/compare/v5.11.3...v5.13.0).

These are the first 10 commits:
- (005a3ba) [Avoid teardown assertion check if test is skipped](https://github.com/seattlerb/minitest/commit/005a3ba42c07d04797e2d00ac2c53e3be127c12f)
- (3f32f02) [Converted all minitest/spec tests over to use _ to avoid dep…](https://github.com/seattlerb/minitest/commit/3f32f0204703afc23549969f5af48d22079f2cf3)
- (879dc1c) [+ Fail gracefully when expectation used outside of `it`.](https://github.com/seattlerb/minitest/commit/879dc1c4c6fb117da15cf212718b2c4e0820a277)
- (e6bc448) [+ Deprecated use of global expectations. To be removed from …](https://github.com/seattlerb/minitest/commit/e6bc4485730403faff6966c1671cf5de72b2d233)
- (1f2b132) [+ Extended Assertions#mu_pp to encoding validity output for …](https://github.com/seattlerb/minitest/commit/1f2b1328f286967926a381d7a34e0eadead0722d)
- (4103a10) [+ Deprecated $N for specifying number of parallel test runne…](https://github.com/seattlerb/minitest/commit/4103a10eb4bb99bc60721b3245ee3a69988a214b)
- (9116a29) [- Fixed Assertions#diff from recalculating if set to nil](https://github.com/seattlerb/minitest/commit/9116a29f9da39c3895472d72735b2a2050b75c94)
- (87f33d7) [Split tests out into test_minitest_assertions.rb](https://github.com/seattlerb/minitest/commit/87f33d7dac890c1ce660baf65a86390509c129bf)
- (b1578db) [Overhauled and sorted test_minitest_assertions.rb in prep fo…](https://github.com/seattlerb/minitest/commit/b1578db8000547c6eee2b5c65ae552d0e00b1dcb)
- (b2eebc2) [+ Changed mu_pp_for_diff to make having both \n and \\n easi…](https://github.com/seattlerb/minitest/commit/b2eebc2d7c492ce0eb11bb88752b841990f9ac92)


</details>